Job start time can be nil, causes panic

https://github.com/openshift/kube-state-metrics/blob/master/collectors/job.go#L181 start time can be nil

```
I0901 18:50:38.616013       1 main.go:254] Starting metrics server: :8080
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1229fe0]

goroutine 309 [running]:
k8s.io/kube-state-metrics/collectors.(*jobCollector).collectJob(0xc4203844d0, 0xc43b992f00, 0x0, 0x0, 0x0, 0x0, 0xc4240c7960, 0x12, 0x0, 0x0, ...)
	/go/src/k8s.io/kube-state-metrics/collectors/job.go:181 +0x210
k8s.io/kube-state-metrics/collectors.(*jobCollector).Collect(0xc4203844d0, 0xc43b992f00)
	/go/src/k8s.io/kube-state-metrics/collectors/job.go:146 +0x133
k8s.io/kube-state-metrics/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func2(0xc43be4fa20, 0xc43b992f00, 0x1d6dca0, 0xc4203844d0)
	/go/src/k8s.io/kube-state-metrics/vendor/github.com/prometheus/client_golang/prometheus/registry.go:433 +0x61
created by k8s.io/kube-state-metrics/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather
	/go/src/k8s.io/kube-state-metrics/vendor/github.com/prometheus/client_golang/prometheus/registry.go:434 +0x27c
```

--- a/collectors/job.go
+++ b/collectors/job.go
@@ -178,7 +178,9 @@ func (jc *jobCollector) collectJob(ch chan<- prometheus.Metric, j v1batch.Job) {
 	addGauge(descJobStatusFailed, float64(j.Status.Failed))
 	addGauge(descJobStatusActive, float64(j.Status.Active))
 
-	addCounter(descJobStatusStartTime, float64(j.Status.StartTime.Unix()))
+	if j.Status.StartTime != nil {
+		addCounter(descJobStatusStartTime, float64(j.Status.StartTime.Unix()))
+	}
 
 	if j.Status.CompletionTime != nil {
 		addCounter(descJobStatusCompletionTime, float64(j.Status.CompletionTime.Unix()))

